src/components/uploader/js/HTML5UploaderSupport.js

```diff
@@ -266,6 +266,26 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             that.events.onBrowse.fire();
         });
 
+        // FLUID-6056 ( https://issues.fluidproject.org/browse/FLUID-6056 )
+        // In IE 11 and MS Edge, < input type="file" > creates an element with
+        // two keyboard focusable parts (textfield and button). When the textfield
+        // is focused ( this happens first when tabbing through elements ), pressing
+        // the "Enter" key triggers a form submission. The workaround implemented
+        // here is to translate the input from the "Enter" key into a click event
+        // on the fileInput so that it will open the OS's file dialog.
+        // Using `navigator.msLaunchUri` to browser detect IE11 and MS Edge, because
+        // it is exclusive to Microsoft browsers for IE 10 and later. This hack
+        // is only needed for IE 11 and MS Edge, and for Firefox this trips the
+        // pop-up blocker.
+        if (navigator.msLaunchUri) {
```

Comment:
@cindyli i was thinking about that, but not sure how feasible it would be in this case because this call is part of several binding methods inside of `the fluid.uploader.bindEventsToFileInput` function. If you could suggest an approach i could give it a try.